rosetta/rosetta/projects/pax/README.md

```diff
@@ -130,23 +130,9 @@ Rosetta Pax containers also provide support for native FP8 through XLA. Enabling
 As of 2/6/2024, CuDNN flash attention was enabled by default via XLA. Divergence has been observed with the GPT 126M model with flash attention enabled. If you observe divergence when running GPT 126M, you can disable flash attention using the following XLA flag: `--set_xla_gpu_enable_cudnn_fmha=False`.
 
 ## XLA Flags
-We recommend setting the following XLA flags when running experiments: 
+The [GPU Performance document](https://github.com/NVIDIA/JAX-Toolbox/blob/main/rosetta/docs/GPU_performance.md) provides a detailed description of the XLA flags that can be set to optimize performance. Additionally, the scripts in `paxml/contrib/gpu/scripts_gpu` automatically set the suggested flags for each model. Please refer to these scripts to find the XLA flags used to reproduce the results documented below.
```

Comment:
Can we use relative links here, so that the link will point to the correct version if it lives off a branch?